modules/heat_conduction/include/fvbcs/FVRadiativeHeatFluxBCBase.h

```diff
@@ -0,0 +1,45 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "FVFluxBC.h"
+
+/**
+ * Boundary condition for radiative heat flux where temperature and the
+ * temperature of a body in radiative heat transfer are specified.
+ */
+class FVRadiativeHeatFluxBCBase : public FVFluxBC
+{
+public:
+  static InputParameters validParams();
+
+  FVRadiativeHeatFluxBCBase(const InputParameters & parameters);
+
+protected:
+  virtual ADReal computeQpResidual();
+
+  /**
+   * qdot = sigma * coeff * (T^4 - Tinf^4 )
+   * sigma: _sigma_stefan_boltzmann
+   * coeff: coefficient()
+   * coefficientBody: cbody
+   * Tinf: temperature of the body irhs
+   */
+  virtual Real coefficient() const = 0;
+
+  /// Stefan-Boltzmann constant
+  const Real _sigma_stefan_boltzmann;
+
+  /// Function describing the temperature of the body irhs
```

Comment:
irhs?

Reply:
Hahaha these abbreviations are copied over from the FE object. Whoever created them loved abbreviations. I think I figured out that `irht` means "in radiative heat transfer". I dunno about this "s" though...

Reply:
It looks like the creator of these classes was... @snschune !

Reply:
Perhaps he can shed some light on `irhs`. Infinite radiative heat surface? That wouldn't go well with the preceding "body" though...

Reply:
Changed to
```c++
  /// Function describing the temperature of the body completely surrounding the surface, e.g. the
  /// temperature of the body we are in radiative heat transfer with
```
after next push